bugzoo/mgr/coverage.py

```diff
@@ -53,12 +54,14 @@ def _from_gcovr_xml_string(self,
         logger = self.__logger.getChild(container.id)
         mgr_ctr = self.__installation.containers
         dir_source = container.bug.source_dir # TODO port
+        resp = mgr_ctr.command(container, "find {}".format(dir_source))
```

Comment:
Would you mind adding the `-type f` option?

Reply:
Also, a short one-line comment explaining what you're doing here would be awesome. (Just so that we remember when we come back to look at it in the future.)